docs/RELEASES.md

```diff
@@ -0,0 +1,422 @@
+11. Update `gh-pages` branch with the new change log
+12. Push `main` branch
+13. Push tags
+14. Push `gh-pages` branch
+
+#### Create release on GitHub
+
+1. Go to https://github.com/thunderbird/thunderbird-android/tags and select the appropriate tag
+2. Click "Create release from tag"
+3. Fill out the form
+   - Click "Generate release notes"
+   - Replace contents under "What's changed" with change log entries
+   - Add GitHub handles in parentheses to change log entries
+   - If necessary, add another entry "Internal changes" (or similar) so people who contributed changes outside of the
+     entries mentioned in the change log can be mentioned via GitHub handle.
+   - Attach the APK
+   - Select "Set as a pre-release"
+   - Click "Publish release"
+
+#### Create release on F-Droid
+
+1. Fetch the latest changes from the _fdroiddata_ repository.
+2. Switch to a new branch in your copy of the _fdroiddata_ repository.
+3. Edit `metadata/com.fsck.k9.yml` to create a new entry for the version you want to release. Usually it's copy & paste
+   of the previous entry and adjusting `versionName`, `versionCode`, and `commit` (use the tag name).
+   Leave `CurrentVersion` and `CurrentVersionCode` unchanged. Those specify which version is the stable/recommended
+   build.
+
+   Example:
+
+   ```yaml
+   - versionName: "${versionName}"
+     versionCode: ${versionCode}
+     commit: "${tagName}"
+     subdir: app-k9mail
+     gradle:
+       - yes
+     scandelete:
+       - build-plugin/build
+   ```
+
+4. Commit the changes. Message: "Update K-9 Mail to $newVersionName (beta)"
+5. Run `fdroid build --latest com.fsck.k9` to build the project using F-Droid's toolchain.
+6. Push the changes to your fork of the _fdroiddata_ repository.
+7. Open a merge request on Gitlab. (The message from the server after the push in the previous step should contain a
+   URL)
+8. Select the _App update_ template and fill it out.
+9. Create merge request and the F-Droid team will do the rest.
+
+#### Create release on Google Play
+
+1. Go to the [Google Play Console](https://play.google.com/console/)
+2. Select the _K-9 Mail_ app
+3. Click on _Open testing_ in the left sidebar
+4. Click on _Create new release_
+5. Upload the APK to _App bundles_
+6. Fill out Release name (e.g. "$versionCode ($versionName)")
+7. Fill out Release notes (copy from `app-metadata/com.fsck.k9/en-US/changelogs/${versionCode}.txt`)
+8. Click _Next_
+9. Review the release
+10. Configure a full rollout for beta versions
+11. On the Publishing overview page, click _Send change for review_
+12. Wait for the review to complete
+13. In case of a rejection, fix the issues and repeat the process
+
+### Release a stable version
+
+When the team decides the `main` branch is stable enough and it's time to release a new stable version, create a new
+maintenance branch (off `main`) using the desired version number with the last two digits dropped followed by `-MAINT`.
```

Comment:
This might also be confusing as we don't use -MAINT in the new system any more. See former comment though.